articles/templates/articles/includes/article_teaser.html

```diff
@@ -14,8 +14,8 @@
             <{{ heading }}><a href="{{ article.url }}">{{ article.title }}</a></{{ heading }}>
             {% if article.authors %}
                 <div class="contributors">By: {% for author in article.authors %}
-                    <div class="contributor"><a href="#">{{ author.first_name }} {{ author.last_name }}</a></div>
-                    {% if not forloop.last %}, {% endif %}{% endfor %}
```

Comment:
You probably still want to have this logic to add commas when there are multiple authors.